src/main/java/storagesystem/viewcontroller/basicStyles.css

```diff
@@ -98,7 +116,28 @@
     -fx-text-fill: #00ff0f;
 }
 
+.confirmButton{
+    -fx-background-radius: 10;
+    -fx-background-color: appButtonColor;
+    -fx-font-size: 20px;
+    -fx-text-fill: white;
+}
+
+.confirmButton:hover{
+    /*TODO Change color to match other buttons*/
```

Comment:
do it

Reply:
No, am waiting for commit from another branch